CardGame.playground/Contents.swift

```diff
@@ -144,5 +202,110 @@ import Foundation
 //: Player 1 drew a 2 of diamonds, player 2 drew a ace of diamonds.
 //: Player 1 wins with 2 of diamonds.
 //: ```
-
-
+let tracker = CardGameTracker()
+let game = HighLow()
+game.delegate = tracker
+
+game.play()
+
+
+print("")
+print("")
+
+
+
+// Just for fun - Here's a different version
+
+// High Low - The Extended Version
+// As the cards are drawn, they are removed from the deck.
+// Scores are kept - 1 point awarded to winner of each round. (No points awarded if it's a tie.)
+// Game is over when all the cards have been drawn (26 rounds). Winner is announced based on final score.
```

Comment:
Love that you went above and beyond. Digging in and pushing past what is required like this is going to take you a long way. I know I've commended you before, but definitely keep it up. You are doing amazing!